app/models/application.rb

```diff
@@ -72,4 +72,38 @@ def cd_enabled?
     key = shortname || fallback_shortname
     Application.cd_statuses.include? key
   end
+
+  def dependency_pull_requests
+    Services.github.search_issues("repo:#{repo} is:pr state:open label:dependencies")
+  end
+
+  def commits
+    Services.github.commits(repo)
+  end
+
+  def tags_by_commit
+    tags.each_with_object({}) do |tag, hash|
+      sha = tag[:commit][:sha]
+      hash[sha] ||= []
+      hash[sha] << tag
+    end
+  end
+
+  def undeployed_commits
+    production_deployment = deployments.last_deploy_to("production")
+
+    comparison = Services.github.compare(
+      repo,
+      production_deployment.version,
+      default_branch,
+    )
+    # The `compare` API shows commits in forward chronological order
+    comparison.commits.reverse + [comparison.base_commit]
+  end
+
+private
+
+  def tags
+    Services.github.tags(repo)
+  end
```

Comment:
Nitpick: I don't see much value in making a proxy method for `Services.github.tags`, especially as we haven't done so for things like `Services.github.compare` on line 95? Makes the diff that little bit harder to read.

Reply:
Good catch - yup especially if we're not using it anywhere else!